haiku-apps/xrs/xrs-1.2.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A simple rhythm station for creating loops"
+DESCRIPTION="XRS is a useful rhythm station for creating your own \
+looping music and could be used to create repeating background music \
+for videos or even a ringtone for your phone."
+HOMEPAGE="https://github.com/HaikuArchives/XRS"
+COPYRIGHT="2001-2003 Giuseppe Gargaro & Andrea Anzani"
+LICENSE="MIT"
```

Comment:
If you do a simple search, you'll find more copyrights and licenses than originally thought ;)

https://github.com/HaikuArchives/XRS/search?q=copyright+path%3AXrsPalm&unscoped_q=copyright+path%3AXrsPalm

Reply:
Thought I found all of them. Thanks for catching me there :)